cpp/src/gandiva/lru_cache.h

```diff
@@ -100,11 +136,58 @@ class LruCache {
     }
   }
 
+  arrow::util::optional<value_type> getObject(const key_type& key) {
+    // lookup value in the cache
+    typename map_type::iterator value_for_key = map_.find(key);
+
+    std::string obj_file_name = "obj-" + std::to_string(key.Hash()) + ".cache";
+    llvm::SmallString<128>obj_cache_file = cache_dir_;
```

Comment:
I think you could extract the `128` magic number to a variable just to make clear why it is choose, or add a comment explain why you choose that number

Reply:
This number was retrieved from llvm docs. It really is necessary to explain it? @anthonylouisbsb

Reply:
@augustoasilva I think you can put at least that information, that the number is retrieved from the LLVM docs.

If you keep just the numbers someone will look at the code and try to guess why you choose that specific number.

Reply:
Good point! I will put a comment there, thanks!